docs/src/tutorial.md

```diff
@@ -213,7 +214,8 @@ hstack(p3, p4)
 | `shape` | `shape_discrete` | `shapekey` |
 | `size` | `size_discrete` | sizekey (tbd) |
 | `linestyle` | `linestyle_discrete` | linekey (tbd) |
-| `group` | `group_discrete` |  |
+| `alpha`  | `alpha_discrete` | alphakey (tbd) |
+| `group`  | `group_discrete` |  |
```

Comment:
i'm not sure the tables of Scales are appropriate in the Tutorial.  they are all enumerated in the Library.  we don't enumerate all Guides or Geoms.  and listing them here disrupts the flow of teaching by example.  i might submit a PR to smooth things out.

Reply:
The tables were introduced in a previous PR (#1245): http://gadflyjl.org/dev/tutorial/.  The relationship between aesthetic, scales and guides is a generally not well understood, and hence these tables need to be in the tutorial.